### PR TITLE
Added Read Command 

### PR DESCRIPTION
**Pull request by junnbang**
Get details of task / customer / driver base on its unique id.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ReadIdCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+        Prefix foundPrefix = getPrefixPresent(argMultimap, PREFIX_TASK, PREFIX_CUSTOMER, PREFIX_DRIVER);
+        int id = ParserUtil.parseId(argMultimap.getValue(foundPrefix).get());
+
+        String className = foundPrefix.getPrefixClass();
+
+        return new ReadIdCommand(className, id);
+    }
+
+    private static int getNoOfPrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
```
Comment (tysng) [inline src/main/java/seedu/address/logic/parser/ReadIdCommandParser.java]: This two chunks of code exist somewhere else in one of the parsers I remember. Consider extracting to `ParserUtil`

**Review by tysg — DISMISSED** [review]

File: src/main/java/seedu/address/logic/parser/ReadIdCommandParser.java (hunk 1 above)
Comment (junnbang) [reply src/main/java/seedu/address/logic/parser/ReadIdCommandParser.java]: solved

File: src/main/java/seedu/address/logic/commands/DoneCommand.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
             + "Example: " + COMMAND_WORD + " 3";
 
     public static final String MESSAGE_INVALID_ID = "Invalid Task id.";
-    public static final String MESSAGE__MARK_TASK_COMPLETED = "Task (%1$s) \n"
```
Comment (ArgVampir) [inline src/main/java/seedu/address/logic/commands/DoneCommand.java]: good catch

**Review by Krit-K — APPROVED** [review]
Nice

**Pull request merged**